mergify_engine/context.py

```diff
@@ -55,6 +55,17 @@
 MARKDOWN_COMMIT_MESSAGE_RE = re.compile(r"^#+ Commit Message ?:?\s*$", re.I)
 
 
+async def retrieve_unverified_commits(
```

Comment:
Since this uses `self`, this is an object instance method. You should put it in the `Context()` class